### PR TITLE
Create RecordShow.java

### PR DESCRIPTION
**Pull request by tulshi123-gupta**
record

### CODE REVIEW

File: RecordShow.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+class RecordShow {
+    // start Recoding after inserting programme name and time id
+    int user_id;
+
+    void recordShow(Scanner inp, int user_id) throws Exception {
+        user_id = this.user_id;
+        PreparedStatement pst;
+        int rs;
+        ResultSet resultSet;
+        SQLQueries sql = new SQLQueries();
```
Comment (rishabhshah2005) [inline RecordShow.java:14]: No need to create new SQLQueries Object in every function
Just declare one in the class and assign it in the constructor

File: RecordShow.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+        String program_routine = "select * from program_routine join channel on program_routine.channel_id=channel.channel_id join  time_slot on program_routine.time_slot_id=time_slot.time_slot_id ";
+        pst = sql.con.prepareStatement(program_routine);
+        resultSet = pst.executeQuery();
+        while (resultSet.next()) {
+            System.out.println(resultSet.getInt("program_id") + "\t"
```
Comment (rishabhshah2005) [inline RecordShow.java:20]: Why are you manually adding spaces to the display functions, 
There are functions to pad strings in Misc.java
```
padAllRight(String[] arr, int L)
padAllRight(String[] arr, int L, String color) 
```

File: RecordShow.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                    + resultSet.getString("program_name") + "\t                  " + resultSet.getTime("start_time")
+                    + "-"
+                    + resultSet.getTime("end_time"));
+        }
+
```
Comment (rishabhshah2005) [inline RecordShow.java:25]: How will the user know the program ID???
There are no functionality to know that
Instead ask which channel you want to record from and the time

**Review by rishabhshah2005 — CHANGES_REQUESTED** [review]
**First of all, You have only uploaded RecordShow.java where are the changes in the HomePage??**
There is no way for me to test this
**Secondly all SQL queries need to be created in SQLQueries.java and not in RecordShows**
Create a main (not psvm) function in RecordShow which contains a while loop with case switch.
Then that class Object will be created at Home page when the correct option is chosen

**_All the other problem have been commented upon in the code itself in the pull request_**

File: RecordShow.java (hunk 3 above)
Comment (tulshi123-gupta) [reply RecordShow.java:25]: here i  have display every program_rouitne with id . and in this i found a mistake as   customer or user have different packages so the can record as it but how to connect idk just now every channenel is available for user

**Pull request closed without merging**